### DISCUSSION TITLE
Release v1.2.5 / 5.57.5

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2045] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build is a test build, it changes two core Sandboxie mechanics.
It reworks the hooking mechanism to enable proper unhooking on dll unload which should solve various compatibility issues
It lets Sandboxie to use custom logon tokens instead of the anonymous one, the option is on by default in the test build and can be controlled with the "SandboxieLogon" setting

This build also adds various UI improvements and a Vintage View Mode which makes Sandman.exe look almost like the classic SbieCtrl.exe

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

### Changed
- improved a few icons

### Fixed
- fixed certificate validation issue

### Added
- added delete button to the recovery window [#2024](https://github.com/sandboxie-plus/Sandboxie/issues/2024)

### Changed
- improved tree selection display

### Fixed
- fixed issues with file panel
- fixed issue with key bindings [#2030](https://github.com/sandboxie-plus/Sandboxie/issues/2030)
- fixed issue wtih RemoveSidName when terminating sbiesvc
- fixed issue with the new hooking mechanism
- fixed BSOD issue with win32k hooks introduced in 1.2.0 [#2035](https://github.com/sandboxie-plus/Sandboxie/issues/2035)
- fixed issue with element11 and electron workaround [#2023](https://github.com/sandboxie-plus/Sandboxie/issues/2023)

### Fixed
- fixed issues with new menu code

### Changed
- reworked frame drawing

### Fixed
- fixed issues with frame-drawing
- fixed issues with the tray and box menu introduced in the last build
- removed focus rectangle from the tree list

### Changed
- refactored menu creation code

### Added
- added Swedish translation to Plus UI (thanks pb1)
- added Vintage View Mode to make SandMan UI look like SbieCtrl
- added alternative tray menu mechanics
- added ability to auto-generate sandbox icons based on the border colour

### Changed
- changed box group icon to a dedicated one
- "browse content" is now available as a side panel in the main window
- animated hourglass icon overlay

### Fixed
- fixed DPI issue on Windows 7
- fixed issue with Software Compatibility tab
- fixed issue with OpenKeyPath introduced in build 1.1.1 [#2006](https://github.com/sandboxie-plus/Sandboxie/issues/2006)

### Added
- re-engineered "SandboxieLogon=y"; it's on by default, as every sandbox gets its own SID now
-- Note: this enforces strict isolation of sandboxes from each other.

### Changed
- reworked hook management, unloaded DLLs are properly unhooked now [#1243](https://github.com/sandboxie-plus/Sandboxie/issues/1243)
- box order is now stored in the Sandboxie-Plus.ini
- improved DPI scaling behaviour

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.2.5'>Release v1.2.5 / 5.57.5</a>.</em>

**Comment by AdminIT-arch** [comment]
> ![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)
> 
> This build is a test build, it changes two core Sandboxie mechanics. It reworks the hooking mechanism to enable proper unhooking on dll unload which should solve various compatibility issues It lets Sandboxie to use custom logon tokens instead of the anonymous one, the option is on by default in the test build and can be controlled with the "SandboxieLogon" setting
> 
> This build also adds various UI improvements and a Vintage View Mode which makes Sandman.exe look almost like the classic SbieCtrl.exe
> 
> #### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.
> #### You can support the project through donations, any help will be greatly appreciated.
> [![](https://camo.githubusercontent.com/510173e5e5d6d8173636c174ec617d79f10e6b66d646329b0b59988ebd6ece37/68747470733a2f2f7777772e70617970616c6f626a656374732e636f6d2f656e5f55532f41542f692f62746e2f62746e5f646f6e61746543435f4c472e676966)](https://xanasoft.com/go.php?to=donate)
> 
> ## [1.2.5 / 5.57.5] - 2022-07-22
> ### Changed
> * improved a few icons
> 
> ### Fixed
> * fixed certificate validation issue
> 
> ## [1.2.4 / 5.57.4] - 2022-07-21
> ### Added
> * added delete button to the recovery window [#2024](https://github.com/sandboxie-plus/Sandboxie/issues/2024)
> 
> ### Changed
> * improved tree selection display
> 
> ### Fixed
> * fixed issues with file panel
> * fixed issue with key bindings [#2030](https://github.com/sandboxie-plus/Sandboxie/issues/2030)
> * fixed issue wtih RemoveSidName when terminating sbiesvc
> * fixed issue with the new hooking mechanism
> * fixed BSOD issue with win32k hooks introduced in 1.2.0 [#2035](https://github.com/sandboxie-plus/Sandboxie/issues/2035)
> * fixed issue with element11 and electron workaround [#2023](https://github.com/sandboxie-plus/Sandboxie/issues/2023)
> 
> ## [1.2.3 / 5.57.3] - 2022-07-13
> ### Fixed
> * fixed issues with new menu code
> 
> ### Changed
> * reworked frame drawing
> 
> ## [1.2.2 / 5.57.2] - 2022-07-13
> ### Fixed
> * fixed issues with frame-drawing
> * fixed issues with the tray and box menu introduced in the last build
> * removed focus rectangle from the tree list
> 
> ### Changed
> * refactored menu creation code
> 
> ## [1.2.1 / 5.57.1] - 2022-07-11
> ### Added
> * added Swedish translation to Plus UI (thanks pb1)
> * added Vintage View Mode to make SandMan UI look like SbieCtrl
> * added alternative tray menu mechanics
> * added ability to auto-generate sandbox icons based on the border colour
> 
> ### Changed
> * changed box group icon to a dedicated one
> * "browse content" is now available as a side panel in the main window
> * animated hourglass icon overlay
> 
> ### Fixed
> * fixed DPI issue on Windows 7
> * fixed issue with Software Compatibility tab
> * fixed issue with OpenKeyPath introduced in build 1.1.1 [#2006](https://github.com/sandboxie-plus/Sandboxie/issues/2006)
> 
> ## [1.2.0 / 5.57.0] - 2022-06-28
> ### Added
> * re-engineered "SandboxieLogon=y"; it's on by default, as every sandbox gets its own SID now
>   -- Note: this enforces strict isolation of sandboxes from each other.
> 
> ### Changed
> * reworked hook management, unloaded DLLs are properly unhooked now [#1243](https://github.com/sandboxie-plus/Sandboxie/issues/1243)
> * box order is now stored in the Sandboxie-Plus.ini
> * improved DPI scaling behaviour
> 
> _This discussion was created from the release [Release v1.2.5 / 5.57.5](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.2.5)._
